google/cloud/spanner_dbapi/connection.py

```diff
@@ -41,28 +52,46 @@ def __handle_update_ddl(self, ddl_statements):
         """
         Run the list of Data Definition Language (DDL) statements on the underlying
         database. Each DDL statement MUST NOT contain a semicolon.
-        Args:
-            ddl_statements: a list of DDL statements, each without a semicolon.
-        Returns:
-            google.api_core.operation.Operation.result()
+
+        :type ddl_statements: list
+        :param ddl_statements: a list of DDL statements, each without a semicolon.
+
+        :rtype: :func:`google.api_core.operation.Operation.result()`
+        :returns: result of the operation
```

Comment:
This probably will break the docs, as you marking function call with `:func:` tags. `result()` function in here will return something - this type should be written in `rtype`